build.gradle

```diff
@@ -35,9 +37,29 @@ repositories {
 // 'api' configuration allows to expose dependencies with 'compile' scope in pom
 // 'implementation' configuration allows to expose dependencies with 'runtime' scope in pom
 dependencies {
+    // iexec
+    implementation "com.iexec.common:iexec-common:$iexecCommonVersion"
+
     implementation platform("org.springframework.boot:spring-boot-dependencies:2.6.14")
```

Comment:
Wouldn't we prefer to have the bom in first place in the `dependencies` block ?
(a kind or priority order which could be BOMs / iExec dependencies / Spring dependencies / ...)